frontends/mit-learn/src/page-components/TestimonialDisplay/AttestantBlock.tsx

```diff
@@ -152,7 +152,7 @@ const AttestantBlock: React.FC<AttestantBlockProps> = ({
         >
           {attestation?.attestant_name}
         </AttestantName>
-        <AttestantTitle variant={variant}>
+        <AttestantTitle variant={variant} color={color}>
```

Comment:
This is inside the `AttestantBlock` component, to which we recently added a `color` prop. `AttestantTitle` should receive the same color.